x-pack/plugins/stack_connectors/server/connector_types/security/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
I added this as a placeholder since the ownership of connectors will be split between cases, E&C and security. The proposed security owned ones are D3 SOAR, Torq and Tines and 2/3 look to have PRs in the pipeline so I wanted to make sure there's a place for them